azure-mgmt-eventhub/src/test/java/com/microsoft/azure/management/eventhub/EventHubTests.java

```diff
@@ -225,15 +225,15 @@ public void canManageEventHubConsumerGroups() throws Exception {
 
         Creatable<EventHubNamespace> namespaceCreatable = eventHubManager.namespaces()
                 .define(namespaceName)
-                .withRegion(REGION)
-                .withNewResourceGroup(RG_NAME);
+                    .withRegion(REGION)
+                    .withNewResourceGroup(RG_NAME);
 
         EventHub eventHub = eventHubManager.eventHubs()
                 .define(eventHubName)
-                .withNewNamespace(namespaceCreatable)
-                .withNewConsumerGroup("grp1")
-                .withNewConsumerGroup("grp2", "metadata111")
-                .create();
+                    .withNewNamespace(namespaceCreatable)
```

Comment:
Are you creating a "parent" namespace as part of the "child" eventHub definition? That will be wrong...

Reply:
This was an ask from event-hub team given that users are more likely to directly start creating event-hub. My question here in general is - do we need to support this if childResource accessor in the current context takes more priority over parent?

Reply:
In this case you already pass a Creatable which means that the user already goes through the process of specifying the required parameters to create the namespace. The user might just call "create()" after that to complete this creation process rather than going through the child resource creation. Also it's not like the two operation can be run in parallel since the event hub creation requires the namespace to be created first.

Reply:
Thinking more about it, I agree with you. Another issue is though we can do something like this for level1 children, it is hard to do the same for level2 onwards. Let's simplify this and provide only one option which is doable in all levels. I will remove it. Thanks!